spec/mongoid/validatable/uniqueness_spec.rb

```diff
@@ -2472,4 +2472,32 @@ class SpanishActor < EuropeanActor
       }.to raise_error(Mongo::Error::OperationFailure)
     end
   end
+
+  describe "i18n" do
+
+    before do
+      Dictionary.validates_uniqueness_of :name
```

Comment:
Please use an existing model field or add a new field to a suitable model.

Reply:
@p-mongo I am following the convention used throughout this spec. Changes to the spec convention should be a separate ticket.

Reply:
This PR is adding wrong code that someone will have to fix in the future. Instead of doing that please add the correct code that won't need to be changed.

Reply:
In my philosophy, within a given code file it is better to be consistently wrong than inconsistently right.

However, I doubt I am going to be able to convince you of this here, so I've done as you've asked.